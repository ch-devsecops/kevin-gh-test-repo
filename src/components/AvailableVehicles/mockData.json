{
  "uid": "1ebf252b-374c-4c65-a977-5a1bf68b2031",
  "componentName": "AvailableVehicles",
  "dataSource": "{1E951413-E986-475B-BA74-D7BA568D167D}",
  "params": {},
  "fields": {
    "data": {
      "value": {
        "anchorId": {},
        "gtmTitle": {},
        "gtmCategory": {},
        "gtmModelName": {},
        "gtmTrimName": {},
        "gtmBodyStyle": {
          "value": "GTM Body Style"
        },
        "gtmInteractionType": {},
        "catalog": {
          "fields": {
            "name": "Honda",
            "models": [
              {
                "items": [
                  {
                    "name": "ODYSSEY",
                    "modelYears": [
                      {
                        "name": "2023",
                        "trims": [
                          {
                            "name": "EX-L",
                            "transmissions": [
                              {
                                "items": [
                                  {
                                    "id": "0971BB011A9A49798362579C17773E51",
                                    "name": "Automatic",
                                    "detKey": {
                                      "value": "10797-Automatic"
                                    },
                                    "detIdentifier": {
                                      "value": "10797"
                                    },
                                    "sotId": {
                                      "value": ""
                                    },
                                    "price": {
                                      "value": "36,000.00"
                                    },
                                    "trim": {
                                      "value": "EX-L"
                                    },
                                    "tooltipText": {
                                      "value": "This is a tooltip"
                                    },
                                    "image": {
                                      "url": "../../../../sample-mock-data/media/img/grey-crv.png"
                                    },
                                    "exteriorColorKey": {
                                      "value": "Grey"
                                    }
                                  }
                                ]
                              }
                            ]
                          },
                          {
                            "name": "TOURING",
                            "transmissions": [
                              {
                                "items": [
                                  {
                                    "id": "0971BB011A9A49798362579C17773E51",
                                    "name": "Automatic",
                                    "detKey": {
                                      "value": "10798-Automatic"
                                    },
                                    "detIdentifier": {
                                      "value": "10798"
                                    },
                                    "sotId": {
                                      "value": ""
                                    },
                                    "price": {
                                      "value": "42,000.00"
                                    },
                                    "trim": {
                                      "value": "Touring"
                                    },
                                    "tooltipText": {
                                      "value": "This is Touring"
                                    },
                                    "image": {
                                      "url": "../../../../sample-mock-data/media/img/blue-accord.png"
                                    },
                                    "exteriorColorKey": {
                                      "value": "blue"
                                    }
                                  }
                                ]
                              }
                            ]
                          },
                          {
                            "name": "BLACK EDITION",
                            "transmissions": [
                              {
                                "items": [
                                  {
                                    "id": "0971BB011A9A49798362579C17773E51",
                                    "name": "Automatic",
                                    "detKey": {
                                      "value": "10799-Automatic"
                                    },
                                    "detIdentifier": {
                                      "value": "10799"
                                    },
                                    "sotId": {
                                      "value": ""
                                    },
                                    "exteriorColorKey": {
                                      "value": "orange"
                                    },
                                    "price": {
                                      "value": "57,550.00"
                                    },
                                    "trim": {
                                      "value": "black-edition"
                                    },
                                    "tooltipText": {
                                      "value": "This is Black Edition"
                                    },
                                    "image": {
                                      "url": "../../../../sample-mock-data/media/img/grey-civic.png"
                                    },
                                    "exteriorColorKey": {
                                      "value": "blue"
                                    }
                                  }
                                ]
                              }
                            ]
                          },
                          {
                            "name": "EX",
                            "transmissions": [
                              {
                                "items": [
                                  {
                                    "id": "0971BB011A9A49798362579C17773E51",
                                    "name": "Automatic",
                                    "detKey": {
                                      "value": "10796-Automatic"
                                    },
                                    "detIdentifier": {
                                      "value": "10796"
                                    },
                                    "sotId": {
                                      "value": ""
                                    },
                                    "price": {
                                      "value": ""
                                    },
                                    "trim": {
                                      "value": "AWD"
                                    },
                                    "tooltipText": {
                                      "value": "This is EX model"
                                    },
                                    "image": {
                                      "url": "../../../../sample-mock-data/media/img/orange-civic.png"
                                    },
                                    "exteriorColorKey": {
                                      "value": "blue"
                                    }
                                  }
                                ]
                              }
                            ]
                          }
                        ]
                      }
                    ]
                  }
                ]
              }
            ]
          }
        }
      }
    },
    "gtmCategory": {
      "id": "86f35331-a03f-4d3e-b8d2-6142e38ad8f3",
      "url": "/sitecore/System/Foundation/common/GTM-Categories/Others",
      "name": "Others",
      "displayName": "Others",
      "fields": {
        "value": {
          "value": "others"
        }
      }
    },
    "gtmTitle": {
      "value": "GTM Title "
    },
    "gtmBodyStyle": {
      "value": "GTM Body Style"
    },
    "gtmInteractionType": {
      "value": "GTM Interaction Type"
    },
    "gtmTrimName": {
      "value": "GTM Trim Name"
    },
    "gtmModelName": {
      "value": "GTM Model Name"
    }
  }
}
