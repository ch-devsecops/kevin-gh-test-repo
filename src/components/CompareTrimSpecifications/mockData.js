// JSS fields coming from Sitecore
export const jssFields = {
  data: {
    value: {
      compareTrimsTitle: {
        value: 'Compare MDX Trims',
      },
      primaryTrim: {
        fields: {
          detIdentifier: {
            value: '10259',
          },
          detKey: {
            value: 'mdx_10259',
          },
        },
      },
      secondaryTrim: {
        fields: {
          detIdentifier: {
            value: '10260',
          },
          detKey: {
            value: 'tech_10260',
          },
        },
      },
      paymentMethod: {
        value: '',
      },
      paymentFrequency: {
        value: 'Monthly',
      },
      modelYear: {
        fields: {
          year: {
            value: 2022,
          },
          model: {
            detKey: {
              value: 'mdx',
            },
          },
          defaultTrim: {
            fields: {
              detIdentifier: {
                value: '10259',
              },
            },
          },
          trims: [
            {
              trimName: {
                value: 'MDX',
              },
              detKey: {
                value: 'mdx_10259',
              },
              detIdentifier: {
                value: '10259',
              },
              compareTrimKeyFeatures: {
                item: {
                  value: '- First item\n- Second item\n- Third item\n- Fourth item\n',
                },
              },
              defaultTransmission: {
                item: {
                  fields: {
                    detKey: {
                      value: '10234-Automatic',
                    },
                    isBuildable: {
                      value: true,
                    },
                  },
                },
              },
              transmissions: {
                item: {
                  fields: {
                    detKey: {
                      value: '10234-Automatic',
                    },
                    isBuildable: {
                      value: true,
                    },
                    defaultExteriorColor: {
                      item: {
                        fields: {
                          color: {
                            fields: {
                              detKey: {
                                value: 'nh_830mlunar_silver_metallic',
                              },
                            },
                          },
                          defaultInteriorColor: {
                            fields: {
                              color: {
                                fields: {
                                  detKey: {
                                    value: 'bk_a_ebony_leatherette_^my22_mdx',
                                  },
                                },
                              },
                            },
                          },
                        },
                      },
                    },
                  },
                },
              },
              primaryThumbnail: {
                item: {
                  value: {
                    src: '/data/media/img/model-card-with-trims/2020_ILX_BAP_ILX_LSM_2.png',
                    alt: 'Primary thumbnail',
                    width: '1440',
                    height: '710',
                  },
                },
              },
            },
            {
              trimName: {
                value: 'Tech',
              },
              detKey: {
                value: 'tech_10260',
              },
              detIdentifier: {
                value: '10260',
              },
              compareTrimKeyFeatures: {
                item: {
                  value: '- First item\n- Second item\n- Third item\n- Fourth item\n',
                },
              },
              defaultTransmission: {
                item: {
                  fields: {
                    detKey: {
                      value: '10235-Automatic',
                    },
                    isBuildable: {
                      value: false,
                    },
                  },
                },
              },
              transmissions: {
                item: {
                  fields: {
                    detKey: {
                      value: '10235-Automatic',
                    },
                    isBuildable: {
                      value: false,
                    },
                    defaultExteriorColor: {
                      item: {
                        fields: {
                          color: {
                            fields: {
                              detKey: {
                                value: '191obsidian_blue_pearl',
                              },
                            },
                          },
                          defaultInteriorColor: {
                            fields: {
                              color: {
                                fields: {
                                  detKey: {
                                    value: 'gr_b_greystone_leather_^my22_mdx',
                                  },
                                },
                              },
                            },
                          },
                        },
                      },
                    },
                  },
                },
              },
              primaryThumbnail: {
                item: {
                  value: {
                    src: '/data/media/img/model-card-with-trims/2020_ILX_BAP_ILX_LSM_2.png',
                    alt: 'Primary thumbnail',
                    width: '1440',
                    height: '710',
                  },
                },
              },
            },
            {
              trimName: {
                value: 'A-SPEC',
              },
              detKey: {
                value: 'a_spec_10261',
              },
              detIdentifier: {
                value: '10261',
              },
              compareTrimKeyFeatures: {
                item: {
                  value: '- First item\n- Second item\n- Third item\n- Fourth item\n',
                },
              },
              defaultTransmission: {
                item: {
                  fields: {
                    detKey: {
                      value: '10236-Automatic',
                    },
                    isBuildable: {
                      value: true,
                    },
                  },
                },
              },
              transmissions: {
                item: {
                  fields: {
                    detKey: {
                      value: '10236-Automatic',
                    },
                    isBuildable: {
                      value: true,
                    },
                    defaultExteriorColor: {
                      item: {
                        fields: {
                          color: {
                            fields: {
                              detKey: {
                                value: 'nh_830mlunar_silver_metallic',
                              },
                            },
                          },
                          defaultInteriorColor: {
                            fields: {
                              color: {
                                fields: {
                                  detKey: {
                                    value: 'wz_z_ebony_leather_with_ultrasuede_^my22_mdx',
                                  },
                                },
                              },
                            },
                          },
                        },
                      },
                    },
                  },
                },
              },
              primaryThumbnail: {
                item: {
                  value: {
                    src: '/data/media/img/model-card-with-trims/2020_ILX_BAP_ILX_LSM_2.png',
                    alt: 'Primary thumbnail',
                    width: '1440',
                    height: '710',
                  },
                },
              },
            },
            {
              trimName: {
                value: 'Platinum Elite',
              },
              detKey: {
                value: 'platinum_elite_10262',
              },
              detIdentifier: {
                value: '10262',
              },
              compareTrimKeyFeatures: {
                item: {
                  value: '- First item\n- Second item\n- Third item\n- Fourth item\n',
                },
              },
              defaultTransmission: {
                item: {
                  fields: {
                    detKey: {
                      value: '10237-Automatic',
                    },
                    isBuildable: {
                      value: true,
                    },
                  },
                },
              },
              transmissions: {
                item: {
                  fields: {
                    detKey: {
                      value: '10237-Automatic',
                    },
                    isBuildable: {
                      value: true,
                    },
                    defaultExteriorColor: {
                      item: {
                        fields: {
                          color: {
                            fields: {
                              detKey: {
                                value: '191obsidian_blue_pearl',
                              },
                            },
                          },
                          defaultInteriorColor: {
                            fields: {
                              color: {
                                fields: {
                                  detKey: {
                                    value: 'gr_b_greystone_leather_^my22_mdx',
                                  },
                                },
                              },
                            },
                          },
                        },
                      },
                    },
                  },
                },
              },
              primaryThumbnail: {
                item: {
                  value: {
                    src: '/data/media/img/model-card-with-trims/2020_ILX_BAP_ILX_LSM_2.png',
                    alt: 'Primary thumbnail',
                    width: '1440',
                    height: '710',
                  },
                },
              },
            },
          ],
        },
      },
    },
  },
};

// Modified trim data passed to CompareTrims React component
export const trims = [
  {
    id: 10259,
    keyFeatures: '* one \n* two \n* three',
    isBuildable: true,
    name: 'MDX',
    primaryThumbnail: {
      src: 'http://via.placeholder.com/300x168/707070?text=mdx',
      alt: 'mdx primary thumbnail',
      width: 300,
      height: 168,
    },
    modelKey: 'mdx',
    modelYear: 2021,
    trimKey: 'mdx_10259',
  },
  {
    id: 10260,
    keyFeatures: '* four \n* five \n* six',
    isBuildable: true,
    name: 'Tech',
    primaryThumbnail: {
      src: 'http://via.placeholder.com/300x168/707070?text=tech',
      alt: 'tech primary thumbnail',
      width: 300,
      height: 168,
    },
    modelKey: 'tech',
    modelYear: 2021,
    trimKey: 'tech_10260',
  },
  {
    id: 10261,
    isBuildable: false,
    name: 'A-SPEC',
    primaryThumbnail: {
      src: 'http://via.placeholder.com/300x168/707070?text=a-spec',
      alt: 'a-spec primary thumbnail',
      width: 300,
      height: 168,
    },
  },
  {
    id: 10262,
    isBuildable: true,
    name: 'Platinum Elite',
    primaryThumbnail: {
      src: 'http://via.placeholder.com/300x168/707070?text=platinum elite',
      alt: 'platinum elite primary thumbnail',
      width: 300,
      height: 168,
    },
  },
];

export default {
  trims,
  jssFields,
};
