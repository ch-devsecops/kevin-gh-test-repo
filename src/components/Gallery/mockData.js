export const strings = {
  Shared: {
    Gallery: {
      galleryModalAria: 'gallery modal',
      exitModalAria: 'exit modal',
      fullscreenModeAria: 'view full screen mode',
      nextImageAria: 'view next image',
      prevImageAria: 'view previous image',
      nextThumbnailPageAria: 'view next thumbnail page',
      prevThumbnailPageAria: 'view previous thumbnail page',
      hideAria: 'hide content',
      hideLabel: 'Hide',
    },
  },
};

export default { strings };
