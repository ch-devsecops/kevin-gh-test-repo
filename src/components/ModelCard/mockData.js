// eslint-disable-next-line import/prefer-default-export
export const ModelCardData = {
  Model: {
    Name: 'Model_Name',
    NameBadge: {
      src: '/data/media/img/model-card/ilx_badge.svg',
      alt: 'model name badge alt text',
    },
    ModelKey: 'mdx',
    modelYear: 2022,
  },
  Trims: {
    SpecialTrimType: 'Available',
    TaglineOverride: 'Performance Luxury Sedan',
    NameBadge: {
      src: '/data/media/img/model-card/type_s.svg',
      alt: 'trim name badge alt text',
    },
  },
  Transmission: {
    IsBuildable: true,
    Name: 'Transmission A',
    TransmissionModelCode: '10201-Automatic',
    HasSpecs: true,
    InteriorColorKey: 'kkebony_leatherette_^my20_ilx',
    ExteriorColorKey: 'bkebony_leatherette_^my20_rdx',
    Trim: 'rdx_10226',
  },
};
