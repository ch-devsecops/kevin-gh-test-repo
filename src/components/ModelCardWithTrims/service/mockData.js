export const Model = {
  NameBadge: '/data/media/img/model-card-with-trims/badge.svg',
  Year: {
    Tagline: 'Compact Sport Sedan',
  },
  ModelKey: 'rdx',
  ModelYear: 2020,
  Trims: [
    {
      TrimName: 'trim name',
      TrimKey: 'rdx_10226',
      TrimThumbnails: {
        PrimaryThumbnail: {
          src: '/data/media/img/model-card-with-trims/primary-thumbnail.png',
          alt: 'Primary thumbnail',
        },
      },
      NameBadge: '/data/media/img/model-card-with-trims/trim-badge.svg',
      DefaultTransmision: '10201-Automatic',
      Transmissions: [
        {
          Name: 'Transmission A',
          TransmissionModelCode: '10201-Automatic',
          IsBuildable: true,
          HasSpecs: true,
          InteriorColorKey: 'kkebony_leatherette_^my20_ilx',
          ExteriorColorKey: 'bkebony_leatherette_^my20_rdx',
        },
        {
          Name: 'Transmission B',
          TransmissionModelCode: 'transmission-b',
          IsBuildable: true,
          HasSpecs: false,
        },
        {
          Name: 'Transmission C',
          TransmissionModelCode: 'transmission-c',
          IsBuildable: false,
          HasSpecs: true,
        },
        {
          Name: 'Transmission D',
          TransmissionModelCode: 'transmission-d',
          IsBuildable: false,
          HasSpecs: false,
        },
      ],
    },
  ],
};

export const strings = {
  Shared: {
    ModelCardWithTrims: {
      exploreModelLabel: 'Explore Model',
      msrpStartingFromLabel: 'MSRP Starting From*',
      offersAvailableLabel: 'Offers Available',
      buildAndPriceLabel: 'Build & Price',
      viewTrimSpecsLabel: 'View Trim Specs',
      priceErrorLabel: 'Unable to load pricing. Please reload the page',
    },
  },
};

export default { Model, strings };
