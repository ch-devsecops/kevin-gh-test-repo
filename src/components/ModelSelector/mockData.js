export const modelSelector = {
  Title: 'Choose Your Model',
  BodyText:
    'Lorem ipsum dolor site amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Lorem ipsum dolor site amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Lorem ipsum dolor site amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
  Models: [
    {
      NameBadge: '/data/media/img/model-selector/badges/ilx.svg',
      Year: {
        Tagline: 'Compact Sport Sedan',
        Trim: {
          PrimaryThumbnail: {
            src: '/data/media/img/model-selector/models/ilx_front.png',
            alt: 'ILX front image',
          },
          SecondaryThumbnail: {
            src: '/data/media/img/model-selector/models/ilx_back.png',
            alt: 'ILX back image',
          },
        },
      },
      ModelKey: 'ilx-111',
    },
    {
      NameBadge: '/data/media/img/model-selector/badges/rlx.svg',
      Year: {
        Tagline: 'Premium Luxury Sedan',
        Trim: {
          PrimaryThumbnail: {
            src: '/data/media/img/model-selector/models/rdx_front.png',
            alt: 'RLX front image',
          },
          SecondaryThumbnail: {
            src: '/data/media/img/model-selector/models/rdx_back.png',
            alt: 'RLX back image',
          },
        },
      },
      ModelKey: 'rlx-111',
    },
    {
      NameBadge: '/data/media/img/model-selector/badges/tlx.svg',
      Year: {
        Tagline: 'Performance Luxury Sedan',
        Trim: {
          PrimaryThumbnail: {
            src: '/data/media/img/model-selector/models/tlx_front.png',
            alt: 'TLX front image',
          },
          SecondaryThumbnail: {
            src: '/data/media/img/model-selector/models/tlx_back.png',
            alt: 'TLX back image',
          },
        },
      },
      ModelKey: 'tlx-111',
    },
    {
      NameBadge: '/data/media/img/model-selector/badges/rdx.svg',
      Year: {
        Tagline: 'Luxury Crossover SUV',
        Trim: {
          PrimaryThumbnail: {
            src: '/data/media/img/model-selector/models/rdx_front.png',
            alt: 'RDX front image',
          },
          SecondaryThumbnail: {
            src: '/data/media/img/model-selector/models/rdx_back.png',
            alt: 'RDX back image',
          },
        },
      },
      ModelKey: 'rdx-111',
    },
    {
      NameBadge: '/data/media/img/model-selector/badges/mdx.svg',
      Year: {
        Tagline: 'Three-Row Luxury SUV',
        Trim: {
          PrimaryThumbnail: {
            src: '/data/media/img/model-selector/models/mdx_front.png',
            alt: 'MDX front image',
          },
          SecondaryThumbnail: {
            src: '/data/media/img/model-selector/models/mdx_back.png',
            alt: 'MDX back image',
          },
        },
      },
      ModelKey: 'mdx-111',
    },
  ],
};

export const accessoriesCtaData = {
  value: {
    href: '/accessories?key=',
    text: 'Explore Accessories',
    linktype: 'internal',
    url: '/accessories?key=',
    title: '',
    class: '',
  },
};
