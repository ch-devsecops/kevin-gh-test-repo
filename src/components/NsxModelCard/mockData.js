export const strings = {
  Shared: {
    NsxModelCard: {
      msrpStartingFromLabel: 'MSRP Starting From*',
      tooltipLabel: 'Price Tooltip Label Dictionary Value',
      priceErrorLabel: 'Unable to load pricing. Please reload the page',
    },
  },
};

export default { strings };
