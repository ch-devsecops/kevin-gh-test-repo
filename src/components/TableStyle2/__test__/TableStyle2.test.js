import React from 'react';
import { jssRender, cleanup, axe } from 'test-utils';

import mockData from '../../../../sample-mock-data/routes/table-style-2/en.json';
import emptyMock from '../../../../sample-mock-data/routes/table-style-2/emptyMock.json';
import TableJSS from '../index';

afterEach(cleanup);

describe('<TableJSS>', () => {
  it('should render correctly', async () => {
    const { container } = jssRender(<TableJSS {...mockData} />);
    expect(container).toMatchSnapshot();
  });

  it('should return null if the content fields is empty', () => {
    const { container } = jssRender(<TableJSS {...emptyMock} />);
    expect(container).toMatchSnapshot();
  });

  it('should have no accessibility violations', async () => {
    const { container } = jssRender(<TableJSS {...mockData} />);
    expect(container).toMatchSnapshot();
    expect(await axe(container)).toHaveNoViolations();
  });
});
