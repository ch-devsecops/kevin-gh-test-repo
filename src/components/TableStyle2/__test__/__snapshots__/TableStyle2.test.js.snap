// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TableJSS> should have no accessibility violations 1`] = `
<div>
  <div
    class="Box-sc-1bsf784-0 cOulSM"
    data-gtm-category="Others"
    data-gtm-title=""
    width="100%"
  >
    <div
      class="Box-sc-1bsf784-0 sc-1k44dfv-0 cxhlwP"
      id=""
    >
      <table
        width="100%"
      >
        <thead
          style="display: block;"
        >
          <tr
            class="Box-sc-1bsf784-0 sc-1k44dfv-0 fxixxS"
          >
            <th
              class="Box-sc-1bsf784-0 sc-1k44dfv-0 loeeBL"
              width="25%"
            >
              <div
                class="sc-1k44dfv-0 klHKZs"
              >
                

                <p>
                  Prefix
                </p>
              </div>
            </th>
            <th
              class="Box-sc-1bsf784-0 sc-1k44dfv-0 dLaUig"
              width="25%"
            >
              <div
                class="sc-1k44dfv-0 klHKZs"
              >
                

                <p>
                  Serial Number Range
                </p>
              </div>
            </th>
            <th
              class="Box-sc-1bsf784-0 sc-1k44dfv-0 dLaUig"
              width="25%"
            >
              <div
                class="sc-1k44dfv-0 klHKZs"
              >
                

                <p>
                  Language
                </p>
              </div>
            </th>
            <th
              class="Box-sc-1bsf784-0 sc-1k44dfv-0 dLaUig"
              width="25%"
            >
              <div
                class="sc-1k44dfv-0 klHKZs"
              >
                

                <p>
                  Manual Download
                </p>
              </div>
            </th>
          </tr>
        </thead>
        <tbody
          class="Box-sc-1bsf784-0 iPRsIc"
        >
          <tr
            class="Box-sc-1bsf784-0 sc-1k44dfv-0 kGSpxe"
          >
            <td
              class="Box-sc-1bsf784-0 sc-1k44dfv-0 frzjuK"
              width="25%"
            >
              <div
                class="sc-1k44dfv-0 diivzi"
                font-family="bold"
              >
                

                <p>
                  GCABE
                </p>
              </div>
            </td>
            <td
              class="Box-sc-1bsf784-0 sc-1k44dfv-0 gQZPxi"
              width="25%"
            >
              <div
                class="sc-1k44dfv-0 diivzi"
                font-family="bold"
              >
                

                <p>
                  1000001 through 9999999
                </p>
              </div>
            </td>
            <td
              class="Box-sc-1bsf784-0 sc-1k44dfv-0 gQZPxi"
              width="25%"
            >
              <div
                class="sc-1k44dfv-0 diivzi"
                font-family="bold"
              >
                

                <p>
                  English
                </p>
              </div>
            </td>
            <td
              class="Box-sc-1bsf784-0 sc-1k44dfv-0 gQZPxi"
              width="25%"
            >
              <a
                class="Link-sc-16wd4a0-0 sc-1k44dfv-0 HZGld"
                cursor="pointer"
                data-gtm-component-type="TableStyle2"
                data-gtm-interaction-type="cta: click"
                data-gtm-title="Download PDF"
                download="/media/img/accordion/accordion-landscape-card.jpg"
                href="/media/img/accordion/accordion-landscape-card.jpg"
              >
                <div
                  class="sc-1k44dfv-0 klHKZs sc-1k44dfv-0 izkjBo"
                >
                  

                  <p>
                    Download PDF
                  </p>
                </div>
                <div
                  class="IconWrapper-lp9q2r-0 bLKGbV"
                />
              </a>
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
</div>
`;

exports[`<TableJSS> should render correctly 1`] = `
<div>
  <div
    class="Box-sc-1bsf784-0 cOulSM"
    data-gtm-category="Others"
    data-gtm-title=""
    width="100%"
  >
    <div
      class="Box-sc-1bsf784-0 sc-1k44dfv-0 cxhlwP"
      id=""
    >
      <table
        width="100%"
      >
        <thead
          style="display: block;"
        >
          <tr
            class="Box-sc-1bsf784-0 sc-1k44dfv-0 fxixxS"
          >
            <th
              class="Box-sc-1bsf784-0 sc-1k44dfv-0 loeeBL"
              width="25%"
            >
              <div
                class="sc-1k44dfv-0 klHKZs"
              >
                

                <p>
                  Prefix
                </p>
              </div>
            </th>
            <th
              class="Box-sc-1bsf784-0 sc-1k44dfv-0 dLaUig"
              width="25%"
            >
              <div
                class="sc-1k44dfv-0 klHKZs"
              >
                

                <p>
                  Serial Number Range
                </p>
              </div>
            </th>
            <th
              class="Box-sc-1bsf784-0 sc-1k44dfv-0 dLaUig"
              width="25%"
            >
              <div
                class="sc-1k44dfv-0 klHKZs"
              >
                

                <p>
                  Language
                </p>
              </div>
            </th>
            <th
              class="Box-sc-1bsf784-0 sc-1k44dfv-0 dLaUig"
              width="25%"
            >
              <div
                class="sc-1k44dfv-0 klHKZs"
              >
                

                <p>
                  Manual Download
                </p>
              </div>
            </th>
          </tr>
        </thead>
        <tbody
          class="Box-sc-1bsf784-0 iPRsIc"
        >
          <tr
            class="Box-sc-1bsf784-0 sc-1k44dfv-0 kGSpxe"
          >
            <td
              class="Box-sc-1bsf784-0 sc-1k44dfv-0 frzjuK"
              width="25%"
            >
              <div
                class="sc-1k44dfv-0 diivzi"
                font-family="bold"
              >
                

                <p>
                  GCABE
                </p>
              </div>
            </td>
            <td
              class="Box-sc-1bsf784-0 sc-1k44dfv-0 gQZPxi"
              width="25%"
            >
              <div
                class="sc-1k44dfv-0 diivzi"
                font-family="bold"
              >
                

                <p>
                  1000001 through 9999999
                </p>
              </div>
            </td>
            <td
              class="Box-sc-1bsf784-0 sc-1k44dfv-0 gQZPxi"
              width="25%"
            >
              <div
                class="sc-1k44dfv-0 diivzi"
                font-family="bold"
              >
                

                <p>
                  English
                </p>
              </div>
            </td>
            <td
              class="Box-sc-1bsf784-0 sc-1k44dfv-0 gQZPxi"
              width="25%"
            >
              <a
                class="Link-sc-16wd4a0-0 sc-1k44dfv-0 HZGld"
                cursor="pointer"
                data-gtm-component-type="TableStyle2"
                data-gtm-interaction-type="cta: click"
                data-gtm-title="Download PDF"
                download="/media/img/accordion/accordion-landscape-card.jpg"
                href="/media/img/accordion/accordion-landscape-card.jpg"
              >
                <div
                  class="sc-1k44dfv-0 klHKZs sc-1k44dfv-0 izkjBo"
                >
                  

                  <p>
                    Download PDF
                  </p>
                </div>
                <div
                  class="IconWrapper-lp9q2r-0 bLKGbV"
                />
              </a>
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
</div>
`;

exports[`<TableJSS> should return null if the content fields is empty 1`] = `<div />`;
